docs/source/tutorials_source/platform/tutorial_aquarium_custom_metadata.py

```diff
@@ -188,8 +188,8 @@
 
     Scatter plot of the tag we named "Fish".
 
-Now, we can use CORESET sampling to diversify this tag and reduce the number of images in it (see `Tutorial 2: Diversify the Sunflowers Dataset` 
-if you're not familiar with subsampling). We use CORESET to create the tag `FewerFish` with only 10 remaining images.
+Now, we can use CORESET selection strategy to diversify this tag and reduce the number of images in it (see `Tutorial 2: Diversify the Sunflowers Dataset`
+if you're not familiar with selection). We use CORESET to create the tag `FewerFish` with only 10 remaining images.
```

Comment:
If you're not familiar with selection sounds strange I would rephrase this. Maybe something like "See tutorial xy for more information on selection strategies".

Reply:
Here too, you gave the 👍 but didn't make any changes